src/steps/os/unix.rs

```diff
@@ -157,6 +157,23 @@ pub fn run_fish_plug(ctx: &ExecutionContext) -> Result<()> {
     ctx.run_type().execute(&fish).args(["-c", "plug update"]).check_run()
 }
 
+/// Upgrades `fundle` and `fundle` plugins.
+///
+/// `fundle` is a package manager for the Fish shell.
+///
+/// See: <https://github.com/danhper/fundle>
+pub fn run_fundle(ctx: &ExecutionContext) -> Result<()> {
+    let fish = require("fish")?;
+    ctx.base_dirs().home_dir().join(".config/fish/fundle").require()?;
+
+    print_separator("fundle");
+
+    ctx.run_type()
+        .execute(&fish)
```

Comment:
Use `.execute(fish)` instead of `.execute(&fish)`
Clippy should show that up/correct that normally.